onchain/.changeset/wise-penguins-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cartesi/rollups": major
+---
+
+ERC20 portal now reverts if the token.transferFrom returns false. Because most developers may assume erc20 transfers to revert if failed. Now inputs will not contain returned bool.
```

Comment:
Please make use of Markdown syntax to format text as code.
Also, we may write "ERC20" and "erc20" in code, but in text, we should use "ERC-20".

```suggestion
The ERC-20 portal now reverts whenever `transferFrom` returns `false`, instead of propagating this information to the DApp back-end. This change aims to prevent developers from accepting failed ERC-20 transfers. As a result, ERC-20 deposit inputs no longer contain the boolean `success` field.
```